chore: BROS-679: Rename `CustomInterface` → `ReactCode`

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/20720933306)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/716
<!-- Follow Merge -->

hlomzik committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20572037381

hlomzik committed:
Remove CustomRegion and related code

It can't be used in LSO anyway

hlomzik committed:
Remove old excess files for CustomInterface

hlomzik committed:
Improve LSO panel about CustomInterface tag being LSE-only

- style for tag name
- React tag name added
- link to LSE docs added

hlomzik committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20720585372

hlomzik committed:
Merge branch 'develop' into 'fb-tag-name-refactor-335e'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20720585372

hlomzik committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20720699822

hlomzik committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20720933306

hlomzik committed:
Merge branch 'develop' into 'fb-tag-name-refactor-335e'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20720933306

hlomzik committed:
Improve LSE-only warning to support ReactCode and tag detection

hlomzik commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20722572861)

hlomzik committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20722572861

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/20754619076
